docs/getting-started/1242-install.mdx

```diff
@@ -156,6 +157,41 @@ If you want to dig into the details of how Dagger works, we recommend you read o
 
 <Button label="How It Works" docPath="f44rm/how-it-works" />
 
+export const Templates = ({ templates }) => {
+  return (
+    <div className={styles.templates}>
+      {templates.map((x, index) => (
+        <TemplateCard
+          key={index}
+          label={x.label}
+          imgFilename={x.imgFilename}
+          url={x.url}
+          dir={x.dir}
+          cmd={x.cmd}
+        />
+      ))}
+    </div>
+  );
+};
+
+export const TemplateCard = ({ label, imgFilename, url, dir, cmd }) => {
+  return (
+    <div className={styles.templateCard}>
+      <div className={styles.templateInfo}>
+        <img src={`/img/getting-started/install-dagger/${imgFilename}`}></img>
+        <a href={url}>{label}</a>
+      </div>
+      <CodeBlock>
+        git clone {url}
+        {`\n`}
+        cd {dir}
+        {`\n`}
+        {cmd}
+      </CodeBlock>
+    </div>
+  );
+};
+
 export const data = {
```

Comment:
Do we import those data somewhere else ?

Reply:
Same as data in how-it-works.mdx